src/data/reducers/assets.test.jsx

```diff
@@ -0,0 +1,290 @@
+import * as reducers from './assets';
+import { assetActions } from '../../data/constants/actionTypes';
+import { assetLoading } from '../../data/constants/loadingTypes';
+import { getAssetAPIAttributeFromDatabaseAttribute } from '../../utils/getAssetsAttributes';
+
+let defaultState;
+
+describe('Assets', () => {
+  describe('assets reducer', () => {
```

Comment:
This refers to the assets state. I know it looks like it's redundant, but it's the assets reducer of the assets.js file.